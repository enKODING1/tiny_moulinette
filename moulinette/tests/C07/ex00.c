#include <stdio.h>

char *ft_strdup(char *src);

int main(void)
{
    printf("%s\n", ft_strdup("Hello World!"));
    printf("%s\n", ft_strdup("Hello 42 Gyeongsan"));
    return 0;
}
