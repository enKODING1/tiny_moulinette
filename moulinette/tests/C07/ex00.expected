Hello World!
Hello 42 Gyeongsan
